website/docs/guides/guides-upgrade.md

```diff
@@ -45,77 +45,124 @@ If you executed the previous step already on your server, then obviously nothing
 
 It is recommended to lock your installation and avoid changes during/after the backup.
 This can either be done by either of the following:
+
 * Lock LanSuite by enabling the corresponding option under "Admin-Page" -> "Common Settings" -> "Lock LanSuite page"
-* Shut down your web server
+* Shut down public access to your web server
 * Move index.php to a different location
 
-### Back up the database
-
-Also multiple variants here:
-* LS XML export ("Admin-Page" -> "Export" -> "XML - Full Database")
-* MySQL commandline dump (Something like `mysqldump --quick -u<ls user> -p <dbname> > exportfile.sql`)
-* PHPMyAdmin - if installed
-* Any tool provided by your hoster
-
-Backing up your database using `mysqldump` is recommended, as both, LanSuite and PHPMyAdmin, may hit server limits on larger installations.
-
-### Backup the current installation
-
-* If you have command line access: `tar cfvz ls_backup.tar.gz <ls_folder>` or `cp -R <ls_folder> <ls_folder>.backup`
-* Download the whole folder with (S)FTP or any other remote access tool you have available
-
 ## Upgrade
 
 ### Copy over files from an old installation
 
-Things of relevance here:
+Replace the new files with the old ones.
+
+A few files and locations are unique to your installation.
+Those should be kept.
+Like:
+
 * The base configuration file from `inc/base/config.php`
 * Anything under `ext_inc`
 * Custom designs from `designs/`
 
 Copy them to the same place on the new installation, overwriting everything there.
 
-### Check file access rights
-
-Depending on the user you did the installation with it may be required to reset the file and folder access rights.
```

Comment:
What is the rationale of removing this?
Unless file operations are done in scope of the webserver user, ownership may be wrong, so may be access rights

Reply:
The installation script checks the file permission and warns the user. Thats why I don't think it should be a manual step. On top of this it doesn't say
- which file permission to check
- what are the required file permission

Hence it is providing limited value.

Reply:
The installation script is not executed as part of the instructions.
So that does not help.
If I remember correctly it also does not tell what should be set, just that permissions are not sufficient.

I do agree with you that it would be better to detail what permissions should be set.
But that does not mean that the information that this may need to be done should be removed alltogether.

Reply:
Now looking deeper at the old upgrade instructions:
> In der Datei inc/base/config.php den Wert von Configured auf 0 zurücksetzen
Nun die Webseite aufrufen. Es erscheint die Installation. Diese ganz normal durchklicken und die Datenbank Zugangsdaten eingeben jedcoh NICHT DAS HÄCKCHEN setzen, dass die DB löscht. Die Installation sollte nun automatisch updaten.

Huh. If that works it covers most of what is needed.